wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+[Half Time wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Half Time"
+[Sudden Death wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Sudden Death"
+[Perfect wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Perfect"
+[Auto wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Auto"
+
+<!-- External links used -->
+[Release b335]: https://osu.ppy.sh/forum/p/26660 "osu! Public Release b335 announcement forum page."
+[osu!academy Ep6]: https://www.youtube.com/watch?v=cyYRl-a5xII "osu!academy Episode 6 youtube link."
+[Match History link]: http://osu.ppy.sh/mp/x
+[modified version link]: https://osu.ppy.sh/forum/t/135726 "ppy's reply on a bug report."
+
+<!-- Image links used -->
+[Beatmaps_peppy image]: /wiki/shared/Beatmaps_peppy.jpg "Official image on beatmap extraction."
+[Multi_lobby image]: ./img/Multi_lobby.jpg "Example of a typical day in the Multi's Lobby"
+[Multi_new_game image]: ./img/Multi_new_game.jpg "Setting up configuration for a New Game"
+[Multi_roomhost image]: ./img/Multi_roomhost.jpg "Example of Host Match Setup"
+[Multi_roomplayer image]: ./img/Multi_roomplayer.jpg "Example of Player Match Setup (With Free Mods on)"
+[Multi_mods_host image]: ./img/Multi_mods_host.jpg "Host (osu!) mods options"
+[Multi_mods_player image]: ./img/Multi_mods_player.jpg "Player (osu!) mods options (With Free Mods on)"
+[Multi_beatmapinfo image]: ./img/Multi_beatmapinfo.jpg "Beatmap information when hovered over."
+[Multi_host_song_change image]: ./img/Multi_host_song_change.jpg "When the Host is changing songs, this bar will become a temporary placeholder."
+[Multi_SS image]: ./img/Multi_SS.jpg "Example of how the Song Selection will look like for host."
+[Multi_Tag_Colour image]: ./img/Multi_tag_colour.jpg "Available colours for the player's turn combo colour"
+[Multi_mh_single image]: ./img/Multi_mh_single.jpg "Example of Match History."
+[Multi_mh_team image]: ./img/Multi_mh_team.jpg "Example of Team Match History."
+
+<!-- Team Mode images -->
+[HTH Interface image]: ./img/Multi_HTH.jpg "Head to Head Interface"
+[Team Vs Interface image]: ./img/Multi_team_vs.jpg "Team Vs Interface"
+[Tag Coop Interface image]: ./img/Multi_tag_co-op.jpg "Tag Coop Interface"
+[Tag Team Vs Interface image]: ./img/Multi_tag_team_vs.jpg "Tag Team Vs Interface"
+[HTH Grade image]: ./img/Multi_grade.jpg "Head to Head grade screen"
+[Team Vs Grade image]: ./img/Multi_team_grade.jpg "Team Vs grade screen"
+[Tag Coop Grade image]: ./img/Multi_co-op_grade.jpg "Team Co-Op grade screen"
+[Tag Team Vs Grade image]: ./img/Multi_tag_team_grade.jpg "Tag Team Vs grade screen"
+
+<!--Title-->
+# Multi
+
+Multi (also known as _Multiplayer_) was implemented on the 03 June 2008 (2008-06-03) under [osu! Public Release b335][Release b335], and is a game mode in which up to sixteen players can compete against each other on a single map decided by the host.
```

Comment:
...`implemented on 03 June 2008 (2008-06-03) under`...
(I would remove the `(2008-06-03)` part since it is formatting for the tables... but tables aren't sortable anyways (and this isn't a table))